platform/web/js/libs/library_godot_audio.d.ts

```diff
@@ -0,0 +1,158 @@
+export type PositionMode = "none" | "2D" | "3D";
```

Comment:
Was this file committed on purpose? I don't understand why we need type definitions for interoperability with TypeScript for an internal library.

Reply:
Yes. It adds typing for easier coding. I'll review the code to make sure that it's still relevant.